app/config/config.neon

```diff
@@ -9,7 +9,6 @@ parameters:
 
   jobConfig:
     storageDir: %appDir%/../job_config
-    humanReadable: true
```

Comment:
Where did this option go?

Reply:
It was used in a single place for an unimportant config of Yaml output. No longer necessary, since yaml_emit is normally human-readable.